src/mlpack/methods/ann/gan/gan.hpp

```diff
@@ -95,47 +93,64 @@ class GAN
   //! Move constructor.
   GAN(GAN&&);
 
+  /**
+   * Prepare the network for the given data.
+   * This function won't actually trigger training process.
+   *
+   * @param trainData The real data.
+   */
+  void ResetData(arma::mat trainData);
```

Comment:
Do you mind to stick with the `predictors/responses` naming pattern?

Reply:
Yes, I changed it.